tests/tck/features/fetch/FetchEdges.intVid.feature

```diff
@@ -71,8 +71,8 @@ Feature: Fetch Int Vid Edges
   Scenario: Fetch prop works with pipeline
     When executing query:
       """
-      GO FROM hash('Boris Diaw') OVER serve YIELD serve._src AS src, serve._dst AS dst
-      | FETCH PROP ON serve $-.src->$-.dst YIELD serve.start_year, serve.end_year
+      GO FROM hash('Boris Diaw') OVER serve YIELD serve._src AS src, serve._dst AS dst |
```

Comment:
Don't modify the unrelated code.

Reply:
my formatting tool will report an error, if `|` at the forefront

Reply:
Maybe you should update format tool.